src/icons/Icon.tsx

```diff
@@ -9,6 +9,8 @@ import {
   StyleProp,
   TextStyle,
   TouchableHighlightProps,
+  RippleBackgroundPropType,
```

Comment:
This is an unnecessary change. There was another PR that addressed this issue so you can undo all the changes in this file

Reply:
Okay, will do the same

Reply:
![Screenshot from 2021-03-14 10-41-01](https://user-images.githubusercontent.com/55053424/111058054-e02c2780-84b1-11eb-80d7-2c324185ef62.png)